app/assets/stylesheets/application.scss

```diff
@@ -32,16 +32,49 @@
  // }
 
 .jumbotran {
-    position: relative;
-    background-color: $pastel-grey;
-    width: 100%;
-    height: 100%;
-    background-size: cover;
-    overflow: hidden;
-
-    border-radius: 25px;
-    border: 2px solid $pastel-orange;
-    padding: 20px;
+  position: relative;
+  background-color: $pastel-grey;
+  width: 100%;
+  height: 100%;
+  background-size: cover;
+  overflow: hidden;
+
+  border-radius: 25px;
+  border: 2px solid $pastel-orange;
+  padding: 20px;
+}
+
+.top {
+  background: url("https://dl.dropboxusercontent.com/u/18396109/gray_background.jpg") no-repeat center center;
```

Comment:
URL `https://dl.dropboxusercontent.com/u/18396109/gray_background.jpg` should not contain protocol or domain